tests/boot/boot_from_pxe.pm

```diff
@@ -98,7 +98,7 @@ sub run {
         send_key "tab";
     }
     if (check_var('BACKEND', 'ipmi')) {
-        $image_path .= "ifcfg=$interface=dhcp4 " unless get_var('NETWORK_INIT_PARAM');
+        $image_path .= "ipv6.disable=1 ifcfg=$interface=dhcp4 " unless get_var('NETWORK_INIT_PARAM');
```

Comment:
Do you have a bugzilla ticket for these cases?... Testing IPMI but disabling features and not linking them to a bug report defeats the purpose of testing baremetal and more important: You seem to be doing this for all of the IPMI machines, regardless of the architecture (Even though the title of your PR says aarch64)... Why?

Reply:
Because I found this problem during integrating aarch64 platform into openQA automation. And this does no harm to IPMI testing, at least, to my best knowledge, unless the SUT does not support IPv4. Please correct me if you have noticed something that really conflicts with this change.  In fact, if you try to ping openqa.suse.de or some other URLs, it only replies to you with its IPv4 address. Any IPv6 site is being used ?

Reply:
@foursixnine You only requested change without any suggestion. What kind of change do you prefer? First of all, I really need to disable IPv6 during installation for some SUTs. So do you agree with introducing another PARAMETER called "IPv6_DISABLE" put into openQA machine settings ? If IPv6_DISABLE=1, then ipv6.dsiable=1 will be appended to kernel. Please fee free to speak out your proposal.

Reply:
I will implement change asap